apps/modernization-ui/src/apps/page-builder/page/management/edit/add-question/search/table/expanded-question.module.scss

```diff
@@ -0,0 +1,36 @@
+@use 'styles/colors';
+
+.expandedQuestion {
+    padding: 1.5rem;
+
+    .buttonBar {
+        padding-bottom: 1.5rem;
+    }
+
+    .content {
+        display: flex;
+        padding: 1.5rem;
+        flex-wrap: wrap;
+        gap: 1.5rem;
+        align-items: flex-start;
+        flex-shrink: 0;
+        border-radius: 0.3125rem;
+        border: 1px solid colors.$base-lighter;
```

Comment:
px to rem

Reply:
removing this border-radius and border styles and leveraging the pre-defined `styles/borders`